### ISSUE TITLE
Open other tex files such as .sty

### ISSUE THREAD

**Issue by aleks223** [Bramas/texiteasy#32]
It's a very nice editor. Just not being able to open other TeX files is weird.

**Comment by Bramas** [comment]
Right, I also had the problem recently ^^. I'll do it and try to deploy it soon.
